Doc/Bib/Year/2019.html

```diff
@@ -867,8 +868,8 @@
 
 <p class='BibEntry'>
 [<span class='BibKeyLink'><a href="http://www.ams.org/mathscinet-getitem?mr=3963995">HIQ19</a></span>]   <b class='BibAuthor'>Hernando, F., Igual, F. D., and Quintana-Ort\'i, G.</b>,
- <i class='BibTitle'>Algorithm  994:  fast  implementations of the Brouwer-Zimmermann algorithm for the computation of the minimum
-                      distance of a random linear code</i>,
+ <i class='BibTitle'>Algorithm 994: fast implementations of the Brouwer-Zimmermann algorithm for the computation of the minimum distance of a
+                      random linear code</i>,
```

Comment:
Note that on the long run, I'd suggest to put all these bib item data into YAML files, too, and then have files like 2019.html be auto-generated. This way we separate the content from its rendering/styling.

Of course that doesn't have to happen in this PR!

Reply:
Yes, that will be a larger scale overhaul of the citation statistics. 

At the moment, the data a drawn from bibtex records, retrieved from MathSciNet. In the meantime, something was changed either there, or in GAPDoc converting bibtex to html...